Cloudinary/Classes/Core/Features/ManagementApi/CLDManagementApi.swift

```diff
@@ -59,6 +66,7 @@ import Foundation
     @discardableResult
     open func rename(_ publicId: String, to: String, overwrite: Bool? = nil, invalidate: Bool? = nil, params: CLDRenameRequestParams? = nil, completionHandler: ((_ result: CLDRenameResult?, _ error: Error?) -> ())? = nil) -> CLDRenameRequest {
         let renameParams = CLDRenameRequestParams(fromPublicId: publicId, toPublicId: to, overwrite: overwrite, invalidate: invalidate)
+        renameParams.setTimeout(from: configuration)
```

Comment:
Can these calls be inside `networkCoordinator.callAction()`? It has an instance of `CLDConfiguration`, we should be able to call `setTimeout()` just once, in there, instead of calling it here on every method (and avoid forgetting to add it when adding new calls later).